### PR TITLE
KEYCLOAK-9712 KEYCLOAK-9911 moved Javascript adater tests and LoginMo…

### PR DESCRIPTION
**Pull request by vramik**
…dulesTest outsite of adapter package

### CODE REVIEW

**Comment by vramik** [comment]
@keycloak-ci-bot test

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/AbstractJavascriptTest.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.keycloak.testsuite.adapter.javascript;
+package org.keycloak.testsuite.javascript;
```
Comment (mhajas) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/AbstractJavascriptTest.java]: @vramik Shouldn't we use Assume.assumeTrue(AUTH_SERVER_SSL_REQUIRED); also for JavascriptTests?

Comment (vramik) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/javascript/AbstractJavascriptTest.java]: @mhajas it depends if we're going to fix the [issue](https://issues.jboss.org/browse/KEYCLOAK-9912) or not :) And it's not ma call, @hmlnarik wdyt?

**Review by slaskawi — CHANGES_REQUESTED** [review]
I believe this approach will not work very well in the pipeline.

The main problem with Javascript Adapter tests is that they run with all App Servers (as you probably noticed, we now use a matrix of jobs for testing adapters, including Wildfly, Jetty, Tomcat etc). This means we execute this tests `n` times (where `n` is the number of App Servers in the matrix). 

The goal of the JIRA is to propose something, to run them only once. The easiest approach is to execute them only with Wildfly. But maybe there's better solution?

**Comment by mhajas** [comment]
> I believe this approach will not work very well in the pipeline.
> 
> The main problem with Javascript Adapter tests is that they run with all App Servers (as you probably noticed, we now use a matrix of jobs for testing adapters, including Wildfly, Jetty, Tomcat etc). This means we execute this tests `n` times (where `n` is the number of App Servers in the matrix).
> 
> The goal of the JIRA is to propose something, to run them only once. The easiest approach is to execute them only with Wildfly. But maybe there's better solution?

@slaskawi This will solve this issue. By this change JavascriptAdapter test will be moved to base testsuite and won't be run as part of any adapter tests pipeline stage. Adapter tests are running using -Dtest=org.keycloak.testsuite.adapter.**.*Test which will not match org.keycloak.testsuite.javascript.

**Comment by slaskawi** [comment]
> This will solve this issue. By this change JavascriptAdapter test will be moved to base testsuite and won't be run as part of any adapter tests pipeline stage. Adapter tests are running using -Dtest=org.keycloak.testsuite.adapter.**.*Test which will not match org.keycloak.testsuite.javascript.

@mhajas @vramik ooh I see! I missed that you moved the tests. Approving.

**Review by slaskawi — APPROVED** [review]

**Review by pdrozd — APPROVED** [review]

**Pull request merged**